wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/model/impl/UserImpl.java

```diff
@@ -22,14 +28,29 @@
  *
  * @author Yevhenii Voevodin
  */
+@Entity(name = "User")
 public class UserImpl implements User {
 
-    private String       id;
-    private String       email;
-    private String       name;
-    private String       password;
+    @Id
+    @Column(name = "id")
```

Comment:
Are you talking about `id` field or about the whole class mapping?
I am wondering that if we change field name the mappings will be also changed which allows migration as well. So it looks more strict for me, and i think it is necessary to name the columns explicitly. Your proposal?